intake_stac/tests/test_catalog.py

```diff
@@ -215,6 +217,23 @@ def test_stac_entry_constructor():
     assert d["metadata"] == item
 
 
+def test_cat_to_geopandas(stac_item_collection_obj):
+    import geopandas as gpd
+
+    cat = StacItemCollection(stac_item_collection_obj)
+    df = cat.to_geopandas()
+    assert isinstance(df, gpd.GeoDataFrame)
```

Comment:
can we also assert that the new df has a few features that we expect (e.g. number or rows, index values, etc.)?

Reply:
Sounds good. I added a few more tests to address this